Use devicectl to install and launch *_application targets on a physical device (#2527)
bazelbuild/rules_apple: doc/tutorials/ios-app.md

### Install the app on a device

You can install the app on a physical device using the bazel run command, just as you would for a simulator:

```bash
bazel run //:iOSApp --ios_multi_cpus=arm64

> ```

Reviewer: We should also recommend users use a `common --flag_alias=ios_device=@rules_apple//apple/build_settings:ios_device` in their `.bazelrc` so they can just do `bazel run //FooApp --ios_device="id"`

Author: Done

The runner will find any available physical device with an OS version higher than `minimum_os_version`.
If you want to specify a particular device, you can specify it like this `--@build_bazel_rules_apple//apple/build_settings:ios_device=<uuid|ecid|serial_number|udid|name|dns_name>`.
To see a list of available devices, use `xcrun devicectl list devices`.

Another way to install the app on the device is to launch Xcode and use the
`Window > Devices and Simulators` command. Select your plugged-in device from the list on the
left, then add the app by clicking the **Add** (plus sign) button under

> "Installed Apps" and selecting the `.ipa` file that you built.

Reviewer: Is this applicable to `rules_apple`?

Author: This universal method for installing an `.ipa` file on a device isn't specific to rules_apple. I've retained it as an alternative to `bazel run` since it was originally included in this documentation.

### Install the app on a device

> You can install the app on a physical device using the bazel run command, just as you would for a simulator:

Reviewer: ```suggestion
You can install the app on a physical device using a bazel run command when targeting a specific device architecture:
```
